release-notes.txt

```diff
@@ -1,3 +1,30 @@
+OpenMDAO Version 1.7.0 Alpha Release Notes
+June 30, 2016
+
+Backwards-Incompatible Change:
+* Change fd_options to be more self consistent, changed name to deriv_options.
+
+Features:
+* NAS access component
+* Derivatives output adds an asterisk to the 'Absolute Error' and 'Relative Error'
+  lines if those lines show a violation greater than some threshold value (1.0E-6 by default).
+* User can now specify scaler and resid_scaler in add_output and add_state metadata.
```

Comment:
User can now specify a scaler on any residual by specifying a resid_scaler during add_state. OpenMDAO see the residual divided by this number.